tests/integration/profiling/rule_based_profiler/test_profiler_user_workflows.py

```diff
@@ -619,7 +619,7 @@ def test_bobby_columnar_table_multi_batch_batches_are_accessible(
 
     # noinspection PyUnresolvedReferences
     pickup_datetime: datetime.datetime = pd.to_datetime(
-        validator_latest.head(n_rows=1)["pickup_datetime"][0]
+        validator_latest.head(n_rows=1).data["pickup_datetime"][0]
```

Comment:
`head` is public. Are you concerned about this api change breaking anyone?

Reply:
This is just being used as a convenience for a test. I made the same change to `validator` after consulting product and Alex.

Reply:
I reverted the change after seeing all of the test changes that were required. I will file a ticket for the changes to `Validator.head()`.